dist/async-test.js

```diff
@@ -1,3 +1,9 @@
+(function (global, factory) {
+    typeof exports === 'object' && typeof module !== 'undefined' ? factory() :
+    typeof define === 'function' && define.amd ? define(factory) :
+    (factory());
+}(this, (function () { 'use strict';
+
 (function () {
```

Comment:
All of the specs have their own closure which auto-detects global. We should remove these?